packages/client/src/generation/generateClient.ts

```diff
@@ -135,6 +135,16 @@ export async function buildClient({
     fileMap['edge.d.ts'] = await TS(edgeTsClient, true)
   }
 
+  if (generator?.previewFeatures.includes('denoDeploy')) {
+    // We use modified version of edge.js in denoDeploy mode.
+    fileMap['edge.js'] = await JS(edgeTsClient, true)
```

Comment:
You could pass another boolean here to signify that this is a Deno+Edge generation. Something like
```suggestion
    fileMap['edge.js'] = await JS(edgeTsClient, true, true)
```

Reply:
Is it better to add the 3rd param to `JS()`? That causes us to pass the option `denoDeploy: boolean` around in many places.

How about adding `denoDeploy?: boolean` option to `TSClientOptions` interface?

Reply:
Yes, that's actually better.